src/backend/distributed/shared_library_init.c

```diff
@@ -437,7 +438,7 @@ _PG_init(void)
 static void
 DoInitialCleanup(void)
 {
-	RepartitionCleanupJobDirectories();
+	CleanupJobCacheDirectory();
```

Comment:
minor: the function comment is outdated?

Reply:
updated